docs/lang/articles/misc/gui.md

```diff
@@ -79,11 +99,158 @@ The data type of `img` must be one of:
 
 - `float64`, range `[0, 1]`
 
+:::note
+
+When using `float32` or `float64` as the data type, `img` entries will be clipped into range [0, 1] for display.
+
+:::
+
+[`gui.get_image()`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=get_image#taichi.misc.gui.GUI.get_image)
+gets the 4-channel (RGBA) image shown in the current GUI system.
+
+[`gui.circle(pos)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=circle#taichi.misc.gui.GUI.circle)
+draws one solid circle.
+
+The color and radius of circles can be further specified with additional parameters.
+
+[`gui.circles(pos)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=circles#taichi.misc.gui.GUI.circles)
+draws solid circles.
+
+The color and radius of circles can be further specified with additional parameters. For a single color, use the `color` parameter.
+For multiple colors, use `palette` and `palette_indices` instead.
+
+For examples:
+```python
+gui.circles(pos, radius=3, color=0x068587)
+```
+draws circles all with radius of 1.5 and blue color positioned at pos array.
```

Comment:
I guess the `radius` argument in gui.circles means `diameter` right? This might confuse people.

Also, we should mention the unit of the digit `radius=3` here, is this number of pixels or something else.

Reply:
Renaming requires modify python API code. I think we should leave this to another PR and focus on docs changing in this one.